### PR TITLE
feat: add frontmatter to override algolia product tags

### PR DESCRIPTION
**Pull request by gabriele-ct**

### CODE REVIEW

File: websites/docs-smoke-test/src/content/components/ai-assistant.mdx [hunk 1]
```diff
@@ -1,5 +1,8 @@
 ---
 title: Ai Assistant
+products:
+  - aiAssistant
+  - additionalProduct
```
Comment (FFawzy) [inline websites/docs-smoke-test/src/content/components/ai-assistant.mdx:5]: can't it be like the planTags property ? `planTags: [plan1, plan2]`

just for consistency

File: websites/documentation/src/content/writing/pages.mdx [hunk 2]
```diff
@@ -56,6 +56,8 @@ We support the following key-value pairs in the frontmatter section:
 
 - `planTags` (array of string): use to indicate what plans are associated to the page.
 
+- `products` (array of string): if defined in a page, it overrides the site wide `products` setting which affects the [algolia search indexer](/configuration/algolia-search#per-site-tags-configuration). This setting is useful for helping indexing correctly specific pages that are part of a site but their focus is on different products.
```
Comment (FFawzy) [inline websites/documentation/src/content/writing/pages.mdx:59]: consider adding usage example in the example already below

**Review by FFawzy — APPROVED** [review]
Works! 🎉 
Thank you 👏 
Very minor nitpicks but good to go

**Comment by FFawzy** [comment]
@melindafekete do you want to take a look at it before we release it ? 

[docs is here](https://docskit-ga-algolia-page-override.commercetools.vercel.app/documentation/writing/pages)


you can see it in action here
https://docskit-ga-algolia-page-override.commercetools.vercel.app/docs-smoke-test/components/ai-assistant

let me know if you need help finding the data, I can help

File: websites/docs-smoke-test/src/content/components/ai-assistant.mdx (hunk 1 above)
Comment (gabriele-ct) [reply websites/docs-smoke-test/src/content/components/ai-assistant.mdx:5]: yes, that's right, better be consistent

**Pull request merged**